### PR TITLE
I4520 : layout change for addon badges

### LINKED ISSUE #4520: Update Addon-header-info-card to match Sketch specs
The Sketch specs for the Addon-header-info-card differ from what we currently have in a number of ways. The layout and formatting of the text is different (which was also reported in issue #4431), and the badges are in a different location.

Sketch has the card looking like:

screenshot 2018-03-07 12 46 58

Whereas the current site looks like:

screenshot 2018-03-07 12 47 45

Duplicates #4431

### PR DESCRIPTION
**Pull request by tsl143**
Fixes #4520 

Before:
screen shot 2018-03-14 at 8 26 49 am

After:
screen shot 2018-03-14 at 8 27 21 am

### CODE REVIEW

File: src/amo/components/AddonBadges/styles.scss [hunk 1]
```diff
@@ -7,19 +7,13 @@
 
   @include respond-to(large) {
     display: flex;
-    flex-flow: row wrap;
+    flex-flow: column;
+    position: absolute;
```
Comment (tofumatt) [inline src/amo/components/AddonBadges/styles.scss]: It seems weird to absolutely position something and have it be a flex container.

Comment (tsl143) [reply src/amo/components/AddonBadges/styles.scss]: yes, I think we can get rid of `display: flex`, if we use `position: absolute`

**Comment by tsl143** [comment]
@tofumatt , in this fix the badges will be stacked vertically, which can overlap with title and author names. So what should be the ideal behavior here? Also max how many badges we are expecting.

**Comment by bobsilverberg** [comment]
Thanks for taking this on, @tsl143. The issue I filed was not just about the badge positioning, but also about some changes to the text and the way it is laid out, some of which is described at https://github.com/mozilla/addons-frontend/issues/4431#issuecomment-369987147. I think it may be fine to only fix the badge positioning in this patch, but we shouldn't close issue #4520 as a result, or we should open a new follow-up issue to address the other changes needed for this card.

**Comment by tsl143** [comment]
@tofumatt got it, I will update this PR.

**Comment by tsl143** [comment]
Used grid to play with position of badges on mobile and desktop.

screen shot 2018-03-15 at 8 07 03 am
screen shot 2018-03-15 at 8 06 51 am

**Comment by tofumatt** [comment]
That’s cool, but seeing them right-aligned on mobile definitely looks weird. Let’s have them aligned to the left on mobile.

- Matt (Sent from mobile)

**Comment by tsl143** [comment]
@tofumatt that will surely look good, in addition, on mobile shall we put icon first, like below?
image

**Comment by tofumatt** [comment]
Yes! Oops, I forgot to even say that but yes that’s awesome. Thanks!

**Comment by tsl143** [comment]
screen shot 2018-03-15 at 11 16 12 pm
screen shot 2018-03-15 at 11 16 01 pm

File: src/ui/components/Badge/styles.scss [hunk 2]
```diff
@@ -9,11 +9,17 @@
...
 
   @include respond-to(medium) {
+    display: flex;
+    flex-direction: row-reverse;
     margin: 0 0 12px;
   }
 
   .Icon {
-    @include margin-start(6px);
+    @include margin(0, 6px, 0, 0);
```
Comment (tofumatt) [inline src/ui/components/Badge/styles.scss]: I didn't know we had this function. I'm guessing it just inverts the left/right for RTL?

Comment (tofumatt) [reply src/ui/components/Badge/styles.scss]: Hah, I see it is. Cool!

**Pull request merged**